lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -258,6 +312,16 @@ class RenderPopover extends RenderShiftedBox {
 
     _contentOffset = _computeContentOffset(arrowLength);
 
+    if (_extendAndClipContentOverArrow) {
+      // Compute the size the child wants to be.
+      final desiredSize = child!.getDryLayout(innerConstraints);
```

Comment:
Why do we run dry layout instead of regular layout? What if we get a child that doesn't support dry layout?

Reply:
It will work with a regular layout. Updated.